model/system/registry.yaml

```diff
@@ -152,6 +152,7 @@ groups:
     brief: "Describes Network attributes"
     attributes:
       - id: system.network.state
+        deprecated: 'Removed, report network connection state with `network.connection.state` attribute'
```

Comment:
Since this attribute is being deprecated here, it should be moved to this file: `/model/system/deprecated/registry-deprecated.yaml`